Sources/ArcGISToolkit/Components/AR/TableTopSceneView.swift

```diff
@@ -168,6 +188,49 @@ public struct TableTopSceneView: View {
         
         // Update extent visualization to the anchor's new geometry.
         planeGeometry.update(from: planeAnchor.geometry)
+        
+        // Set help text when plane visualization is updated.
+        helpText = .planeFound
+    }
+    
+    /// Updates the help text to guide the user through an AR experience using the AR session's camera tracking status.
+    /// - Parameter trackingState: The camera's tracking status.
+    private func updateHelpText(for trackingState: ARCamera.TrackingState) {
+        guard !initialTransformationIsSet else {
+            helpText = ""
+            return
+        }
+        
+        switch trackingState {
+        case .normal:
+            helpText = .moveDevice
+        case .notAvailable:
+            helpText = .locationUnavailable
+        case .limited(let reason):
+            switch reason {
+            case .excessiveMotion:
+                helpText = .excessiveMotion
+            case .initializing:
+                helpText = .moveDevice
+            case .insufficientFeatures:
+                helpText = .insufficentFeatures
+            case .relocalizing:
+                // This case will not occur since the AR session delegate
+                // does not implement relocalization support.
+                break
+            default:
+                break
```

Comment:
you might want to set it to empty here otherwise it might be incorrect if a new tracking state is added